### DISCUSSION TITLE
groupRowsBy not working when using paginator and moving to second page

### DISCUSSION THREAD

**Discussion by maxp121** [primefaces/community#1675] [PrimeNG]
Hello everyone,

i am using  rowGroupMode='subheader' [groupRowsBy]="'object.' + componentData.rowObjectPropertyNameForGrouping"
to group rows with subheader.
i am using PrimeNG version 17.6.0;

this works very well when the table shows the first page however when i move to the next page the rows loose the group by and each row is presented under a separate header.

this how it looks on first page:
![image](https://github.com/primefaces/community/assets/140953453/fc8807f9-7b80-4b57-8d59-b0f44a8e6cf3)

this is the next page:
![image](https://github.com/primefaces/community/assets/140953453/8b5f69b6-b667-44c0-9a6a-74597559b5ff)

this was the same behavior before upgrading to latest PrimeNG version.

**Comment by cetincakiroglu** [comment]
Hi,

Could you please share a stackblitz example so we can identify the issue?

**Comment by cetincakiroglu** [comment]
Hi,

Could you please try it with the 17.14.0, we've merged a PR that could potentially fixed the issue.
